decrepit/xts/xts_test.cc

```diff
@@ -206,6 +208,245 @@ static const XTSTestCase kXTSTestCases[] = {
         "dfce93b6f82ae22423274e58a0821cc5502e2d0ab4585e94de6975be5e0b4efce51cd3"
         "e70c25a1fbbbd609d273ad5b0d59631c531f6a0a57b9",
     },
+    // https://github.com/BrianGladman/modes/blob/master/testvals/xts.6#L313
```

Comment:
just curious, how did you choose the specific test vectors from that github repo to include here?

Reply:
I went through the many test vector files under `testvals` choosing the ones that had 256-bit key with a plaintext length that I was looking for.

Reply:
thanks!